.github/workflows/test.yml

```diff
@@ -145,6 +145,28 @@ jobs:
         rubygems: 3.2.3
     - run: gem --version | grep -F "3.3.3"
 
+  testMajorBundlerVersion:
+    name: "Test with a major Bundler version"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ./
+      with:
+        ruby-version: 2.6
+        bundler: 2
+    - run: bundle --version | grep "Bundler version 2\.\d+.\d+"
```

Comment:
I think `grep` without any flag doesn't understand `\d+`.
It' seems to work with `grep -P`:
```
$ echo 2.2.32 | grep -P "2\.2\.\d+"
2.2.32
```